README.md

```diff
@@ -8,11 +8,16 @@ Save the following code as `Format.yml` in the `.github/workflows/` directory in
 name: Format suggestions
 on:
   pull_request:
+    # this argument is not required if you don't use the `suggestion-label` input
+    types: [ opened, reopened, synchronize, labeled, unlabeled ]
 jobs:
   code-style:
     runs-on: ubuntu-latest
     steps:
       - uses: julia-actions/julia-format@v2
+        with:
+          version: 1 # set this to e.g. 1.0.54 if you need a stable version
```

Comment:
```suggestion
          version: "1" # Set `version` to "1.0.54" if you need to use JuliaFormatter.jl v1.0.54 (default: "1")
```
- I think it would be better to encourage using `""` to distinguish `1.1` and `1.10`.
- There are many versions in this context (JuliaFormatter, julia-format and Julia itself), so adding "JuliaFomatter" in the comment would be helpful.